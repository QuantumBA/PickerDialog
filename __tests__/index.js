import {mount} from 'enzyme'
import React from 'react'

import Picker from '..'


test('empty, no props', () => {
  const wrapper = mount(<Picker/>)

  expect(wrapper).toMatchSnapshot()
})
