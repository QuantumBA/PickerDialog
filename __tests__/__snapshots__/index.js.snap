// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`empty, no props 1`] = `
<PickerDialog>
  <PickerIOSDialog
    mode="dialog"
  >
    <Component
      style={
        Array [
          Object {
            "flex": 1,
            "marginTop": 60,
          },
          undefined,
        ]
      }
    >
      <react-native-mock>
        <TouchableHighlight
          onPress={[Function]}
          style={
            Object {
              "marginBottom": 25,
              "marginTop": 25,
            }
          }
          underlayColor="transparent"
        >
          <react-native-mock>
            <Component
              style={
                Object {
                  "textAlign": "center",
                }
              }
            >
              <react-native-mock />
            </Component>
          </react-native-mock>
        </TouchableHighlight>
      </react-native-mock>
    </Component>
  </PickerIOSDialog>
</PickerDialog>
`;
